client/src/components/frontend/Utility/ShareBar.js

```diff
@@ -53,7 +58,9 @@ class ShareBar extends Component {
 
     return (
       <nav className="share-nav-primary">
-        <span>Share</span>
+        { this.props.label ?
```

Comment:
Because this change is unrelated to the annotation work, it should be broken out into a separate commit.